script.recap/resources/skins/Default/720p/script-recap-window.xml

```diff
@@ -0,0 +1,197 @@
+            <onleft>302</onleft>
+            <onright>302</onright>
+        </control>
+		<control type="image" id="204">
+			<description>Slideshow Image</description>
+			<posx>40</posx>
+			<posy>250</posy>
+			<width>720</width>
+			<height>385</height>
+			<aspectratio>keep</aspectratio>
+			<texture></texture>
+		</control>
+        <control type="textbox" id="205">
+            <description>Slideshow Caption</description>
+            <posx>50</posx>
+            <posy>551</posy>
+            <width>700</width>
+            <height>100</height>
+            <align>center</align>
+            <colordiffuse>FFFFFFFF</colordiffuse>
+            <textcolor>CCFFFFFF</textcolor>
+			<font>font13</font>
+			<shadowcolor>CC000000</shadowcolor>
+        </control>
+        <control type="button" id="305">
+            <description>Previous Page Button</description>
+            <posx>40</posx>
+            <posy>60r</posy>
+            <width>100</width>
+            <height>35</height>
+            <align>center</align>
+            <label>&lt;</label>
+            <texturenofocus border="5">button-nofocus.png</texturenofocus>
+            <texturefocus border="5">button-focus.png</texturefocus>
+            <colordiffuse>FF43C6DB</colordiffuse>
+			<textcolor>ffffffff</textcolor>		<!-- white -->
+            <onright>302</onright>
+            <onleft>304</onleft>
+            <onup>203</onup>
+            <ondown>203</ondown>
+        </control>
+        <control type="button" id="302">
+            <description>Close Button</description>
+            <posx>275</posx>
+            <posy>60r</posy>
+            <width>250</width>
+            <height>35</height>
+            <align>center</align>
+            <label>$ADDON[script.recap 32006]</label>
+            <texturenofocus border="5">button-nofocus.png</texturenofocus>
+            <texturefocus border="5">button-focus.png</texturefocus>
+			<textcolor>ffffffff</textcolor>		<!-- white -->
+            <onright>304</onright>
+            <onleft>305</onleft>
+            <onup>203</onup>
+            <ondown>203</ondown>
+        </control>
+        <control type="button" id="304">
+            <description>Next Page Button</description>
+            <posx>620r</posx>
+            <posy>60r</posy>
+            <width>100</width>
+            <height>35</height>
+            <align>center</align>
+            <label>&gt;</label>
+            <texturenofocus border="5">button-nofocus.png</texturenofocus>
+            <texturefocus border="5">button-focus.png</texturefocus>
+			<textcolor>ffffffff</textcolor>		<!-- white -->
+            <onright>305</onright>
+            <onleft>302</onleft>
+            <onup>203</onup>
+            <ondown>203</ondown>
+        </control>
+		<!-- The following are a list of cached images -->
+		<control type="image" id="401"><posx>40</posx><posy>636</posy><width>12</width><height>5</height><texture background="true"></texture></control>
```

Comment:
If it switched to a Container, I'm not sure I'd be able to access if from Python via the API would I? I'd need that to be able to set the images.